results/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/prometheus/procfs v0.0.8 // indirect
 	github.com/stretchr/testify v1.5.1 // indirect
 	github.com/tektoncd/pipeline v0.12.0
+	github.com/yuege01/experimental v0.7.0 // indirect
```

Comment:
You generally shouldn't need to do this. Local references should generally resolve themselves, otherwise you should use a `replace` to alias to your local copy.

Reply:
It actually is automatically generated by protoc, I am not sure how to remove it : )